benchmarks/text_hybrid_scoreboard/test_sqlite_fts5_bench.py

```
@@ -0,0 +1,49 @@
+                    "--queries",
+                    "2",
+                    "--top-k",
+                    "5",
+                    "--out",
+                    str(out),
+                ],
+                check=False,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                text=True,
+            )
```

Ruff (0.15.15)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)